### ISSUE TITLE
Program run from Executables menu do not see properly merged directories

### ISSUE THREAD

**Issue by whraven** [Modorganizer2/modorganizer#374]
# The problem

When running a program from the Executables menu, only the first directory encountered is seen rather than a merge of all directories with the same structure. Data tab, however, shows the merged directory structure properly.
 
# Environment

Mod Organizer 2.1.3 under Windows 10 Home, version 1803.

# Details

First noticed with Bodyslide. When run from the Executables menu, it only sees the first loaded CalienteTools/BodySlide/Slider* (SliderGroups, SliderSets, SliderPresets) directory instead of a merged view from all loaded mods. Examining the directory structure using the data tab, however, shows a properly merged view. If Bodyslide is run by right-clicking in the Data tab and selecting Open/Execute, it does see a properly merged directory structure.

At this time, I do not know if this issue is restricted to Bodyslide, or applies to other programs as well.

**Comment by whraven** [comment]
Problem is a result of switching from MO 1.3.14 to MO 2.1.3 and using the original configuration .ini file. If Bodyslide is removed from the menu, and re-configured making certain the "Start In" directory is correct, then it works properly.

**Comment by Silarn** [comment]
Just FYI there should be no problem using the default start in setting assuming it's installed as a mod and enabled or in the data directory.

**Comment by whraven** [comment]
I agree. I think it may be something peculiar to how I set up the migration. I linked the appropriate MO2 directories to the MO1 ones so that both would see the same profiles, mods, and downloads. Bodyslide did not see the same dir structure that the Data tab showed. I deleted the Bodyslide executable configuration, re-entered it, configuring both the executable and the starting directory, checked that both were correct, and completed the operation. Bodyslide still did not see the same directory structure as the Data tab showed. Re-checking, I see that instead of the starting directory I entered, it showed the MO1 version of the directory. However, if I right-clicked on the Bodyslide executable in the Data tab itself and used "add as Executable", both the executable and starting directory were configured just as I had entered them, but when running Bodyslide, the starting directory remained stable and Bodyslide had the same view of the directory structure as the Data tab. I'm rather puzzled by this difference in behaviour, but at this point I'm just chalking it up to the way I am handling the migration. I still have an issue (mentioned on Discord, I haven't entered it as an issue yet) with Skyrim reading .jslot files differently between MO1 and MO2, but I don't think it's related to this.
